src/sssom/writers.py

```diff
@@ -436,7 +436,8 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
 def to_json(msdf: MappingSetDataFrame) -> JsonObj:
     """Convert a mapping set dataframe to a JSON object."""
     doc = to_mapping_set_document(msdf)
-    data = JSONDumper().dumps(doc.mapping_set, contexts={"@context": doc.prefix_map})
```

Comment:
I see right here there is directly an `@context`, what is the problem?

Reply:
There is a lot of stuff missing in that context, pertaining to SSSOM internal elements. Look at what Anita posted how it was previously.

Reply:
fixed in eec2faa but we need a proper unit test